api/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2013 Medic Mobile, 501(c)(3)  <hello@medicmobile.org>
+Copyright 2013-2021 Medic Mobile, Inc 501(c)(3) <hello@medic.org>
```

Comment:
should this still read Medic Mobile?

Reply:
Yes, copyrights or licenses (legal-ish) needs to stay as "Medic Mobile, Inc".

Reply:
Okay.